membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -219,6 +221,14 @@ object AccountDetails {
       case _ => plan.productName
     }
 
+  def getMMAProductKey(catalog: Catalog, plan: RatePlan): String =
+    plan.product(catalog) match {
+      case _: Product.Newspaper if plan.name(catalog) == "Sunday" => plan.productName + " - Observer"
+      case _: Product.Newspaper if plan.hasChargeType(catalog, Digipack) => plan.productName + " + Digital"
+      // TODO if it's the new tier 3 (digital only) return a different key (later)
```

Comment:
this will happen in a later PR once the catalog has it added and we know the chargeType or rate plan name depending on how we want to detect it